src/main/java/duke/ui/Ui.java

```diff
@@ -0,0 +1,88 @@
+package duke.ui;
+
+import duke.events.Task;
+
+import java.util.List;
+
+/**
+ * Class that deals with interactions with the user
+ * Has static methods and does not need to be initialized
+ */
+public class Ui {
+
+    private static String logo = " ____        _        \n"
+            + "|  _ \\ _   _| | _____ \n"
+            + "| | | | | | | |/ / _  \n"
+            + "| |_| | |_| |   <  __/ \n"
+            + "|____/ \\__,_|_|\\_\\___|\n";
+
+
+    public static final String ENDING_MESSAGE = "That's all? Hope to see you again soon :)";
+
+    public static String welcomeMessage() {
+        String message = "";
+        message += "Hello from\n" + logo;
+        System.out.println("Hello from\n" + logo);
+        message += "Hello! I'm the new and improved Duke \n"
+                + "How may I be of assistance today?";
+        System.out.println("Hello! I'm Duke \n"
+                + "How may I be of assistance today?");
+        return message;
+    }
+
+    /**
+     * Used when index is out of bounds for add/delete
+     * @return formatted Out of bounds message string
+     */
+    public static String outOfBoundsMessage(int upperLimit, int index) {
```

Comment:
Maybe this method name should be a verb instead?

Reply:
Hey good catch there thank you! Will change the Javadocs and method spacing issues as well